.github/actions/orca/orca-tutorial-NCF-ray/action.yml

```diff
@@ -0,0 +1,35 @@
+name: 'Run Orca Tutorial NCF Ray'
+description: 'Run Orca Tutorial NCF Ray'
+runs:
+  using: "composite"
+  steps:
+    - name: Run Test
+      shell: bash
+      run: |
+        source activate py37
+
+        export SPARK_LOCAL_HOSTNAME=localhost
+        bash python/dev/release_default_linux_spark313.sh default false false false -Ddata-store-url=$HTTP_URI -U
+        unset SPARK_HOME
+        
+        # install dlib
+        pip install -i https://pypi.org/simple python/dllib/src/dist/bigdl_dllib*-py3-none-manylinux1_x86_64.whl
+        # install orca
+        pip install -i https://pypi.org/simple python/orca/src/dist/bigdl_orca*-py3-none-manylinux1_x86_64.whl
+        # install requires
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} bigdl-orca-spark3[ray]
```

Comment:
Can we just install the package `bigdl-orca-spark3` (or `bigdl-orca-spark3[ray]`) without build the whole bigdl? The environment can be closer to **user daily usage**.

Reply:
Theoretically for the pr validation, we need to manually build and install so that the immediate changes would take impact.
For the nightly build, we need to pip install to be closer to user daily usage.